apps/epl/src/epl.erl

```diff
@@ -23,24 +27,75 @@
 %% ===================================================================
 %% API functions
 %% ===================================================================
+
+%% Lookup for given Key in epl_priv ets
+-spec lookup(Key :: term()) -> [tuple()].
 lookup(Key) ->
     ets:lookup(epl_priv, Key).
 
+%% Add calling process to every epl_tracers' subscribers list
```

Comment:
Use the `Edoc` for documenting the public functions (http://erlang.org/doc/apps/edoc/chapter.html). For example the `@doc` tag and the `' for variables.